packages/preview/typxidian/0.1.1/src/dependencies.typ

```diff
@@ -1,8 +1,10 @@
 #import "@preview/cetz:0.4.2" // drawing inspired by tikz
+#import "@preview/cetz-plot:0.1.2"
```

Comment:
cetz-plot 0.1.3 was just released to ensure compatibility with cetz 0.4.2 and later, you should probably update.